config/e2e/helm-operator-under-test.yaml

```diff
@@ -20,6 +20,7 @@ podAnnotations:
 tracing:
   enabled: true
   config:
+    ELASTIC_APM_SERVER_URL: null
```

Comment:
We have ha default of localhost in `values.yaml` that takes precedence otherwise.

Reply:
Yes, that make sense. It was only added to the values.yaml to act as an example of how to set different tracing related environment variables.